packages/nestjs/README.md

```diff
@@ -0,0 +1,10 @@
+<p align="center">
+  <a href="https://sentry.io/?utm_source=github&utm_medium=logo" target="_blank">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-wordmark-dark-280x84.png" alt="Sentry" width="280" height="84">
+  </a>
+</p>
+
+# Official Sentry SDK for NestJS
+
+This SDK is considered **experimental and in an alpha state**. It may experience breaking changes. Please reach out on
+[GitHub](https://github.com/getsentry/sentry-javascript/issues/new/choose) if you have any feedback or concerns.
```

Comment:
I think we should flesh out the readme a bit more. Ideally we get it to a similar state as the readme of the `@sentry/node` package with basic setup instructions and how to use. Once we add more features to this package we can update this readme again.